src/main/java/com/zufar/icedlatte/review/converter/ProductRatingDtoConverter.java

```diff
@@ -13,4 +16,32 @@ public ProductRatingDto convertToDto(ProductRating productRating) {
                 productRating.getProductRating()
         );
     }
+
+    public RatingMap convertToRatingMap(List<Object[]> listOfMappings) {
+        var map = new RatingMap();
```

Comment:
RatingMap is deprecated

Reply:
Yes, but what alternative can be used here?